packages/plugin-nm/sources/NodeModulesLinker.ts

```diff
@@ -1270,6 +1279,8 @@ async function persistNodeModules(preinstallState: InstallState, installState: N
   const reportedProgress = report.reportProgress(progress);
   const nmModeSetting = project.configuration.get(`nmMode`);
   const nmMode = {value: nmModeSetting};
+  const nmFolderLinkModeSetting = project.configuration.get(`nmFolderLinkMode`);
+  const nmFolderLinkMode = {value: nmFolderLinkModeSetting};
```

Comment:
There is no need to make `nmFolderLinkMode` an object with this structure. Don't look at `nmMode` since, in case of `nmMode` it might be dynamically changed during code execution when certain factors become true. In case of `nmFolderLinkMode` - it never changes, so need to wrap it into an object.

Reply:
Thanks, was wondering why nmMode used this structure and just copied the patterns I saw in the code. 

I'll update the PR.